src/main/fc/rc_adjustments.c

```diff
@@ -656,6 +662,19 @@ static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, ui
         }
 #endif
         break;
+    case ADJUSTMENT_SIMPLIFIED_MASTER_MULTIPLIER:
+    {
+        static uint8_t lastPosition = 255;
+
+        if (position != lastPosition) {
+            float newMultiplier = 10.0f + (position * 1.9f);
```

Comment:
This way it is more explicit. Also, there is no need to use float
```suggestion
            int newMultiplier = scaleRange(position, 0, 100, 10, 200);
```